### ISSUE TITLE
Why do you need extra libs?

### ISSUE THREAD

**Issue by vinayakkulkarni** [ElementUI/element-in-laravel-starter#33]
in the package.json, do you really need bootstrap, jquery, lodash etc. libs ?
can you clean out all the extra dependencies and have just Element-UI + Vue + Laravel.

**Comment by nicolasbeauvais** [comment]
This is a copy of the original Laravel package.json: 
https://github.com/laravel/laravel/blob/master/package.json

I think it makes sense to keep the packages required by the default Laravel install.

**Comment by csvwolf** [comment]
I agree with @nicolasbeauvais . In fact some backend developers want to use jQuery & Vue as a combo.

**Comment by rushairer** [comment]
php artisan preset none
